ga4gh/backend.py

```diff
@@ -176,6 +176,20 @@ def _getEnd(cls, readAlignment):
             len(readAlignment.alignedSequence))
 
 
+class ReadGroupSetIntervalIterator(ReadsIntervalIterator):
+    """
+    An interval iterator for reads, for an entire ReadGroupSet
+    """
+    def __init__(self, request, parentContainer, reference):
+        self._reference = reference
```

Comment:
I suppose it is!  I thought that passing in a different object  would have different implications down the line, but I guess not if the read-accessor methods are named the same and have the same prototype.